catalog/ee-pkurelab/projects_ee-pkurelab_assets_LHScat_v1.jsonnet

```diff
@@ -0,0 +1,105 @@
+
+{
+  'gee:skip_indexing': true,
+  
+  stac_version: ee_const.stac_version,
+  type: ee_const.stac_type.collection,
+  stac_extensions: [
+    ee_const.ext_eo,
+    ee_const.ext_sci,
+  ],
+  id: id,
+  title: 'LHScat dataset',
+  'gee:type': ee_const.gee_type.image_collection,
+  description: |||
+    The LHScat dataset is the first long-term (since 1992), high-resolution (∼8.9 km) 
+    monthly satellite radar backscatter dataset over global land areas. It integrated 
+    signals from the European Remote Sensing satellite (ERS; 1992–2001; C-band; 5.3 GHz),
+    Quick Scatterometer (QSCAT, 1999–2009; Ku-band; 13.4 GHz), and the Advanced 
+    SCATterometer (ASCAT; since 2007; C-band; 5.255 GHz). The 6-year data gap between 
+    C-band ERS and ASCAT was filled by modelling a substitute C-band signal during 1999–2009 
+    from Ku-band QSCAT signals and climatic information. The quality of the merged radar 
+    signal was assessed by computing the Pearson r and root mean square error (RMSE) 
+    between the C-band and the corrected Ku-band signals in the overlapping years (1999–2001 
+    and 2007–2009). The components of LHScat dataset are as follows:
+    Monthly radar signals: The dataset is available from April 1992 to August 2021, with a spatial resolution of 8.9 km. 
+    To facilitate storage, the radar signals were scaled by a factor of 100 and converted to unsigned Int16.
+    Pearsonr: Pearsonr between the monthly C-band and corrected Ku-band signals in the overlapping years.
+    RMSE: RMSE between the monthly C-band and corrected Ku-band signals in the overlapping years.
+    Satellite: The satellite source of the signal. The values are as follows:1. ERS; 2. ERS and QSCAT; 3. QSCAT; 4. ASCAT and QSCAT; 5. ASCAT .
+    Since the signals were merged from several sensors, the Pearsonr and RMSE values can be used to select pixels with high data merging quality. 
+    For more detailed information, please refer to the paper describing LHScat dataset(https://essd.copernicus.org/articles/15/1577/2023/).
+  |||,
+  license: license.id,
+  links: ee.standardLinks(subdir, id),
+  'gee:categories': ['satellite-imagery'],
+  keywords: [
+    'radar',
+    'C-bands',
+    'Ku-band',
+    'QSCAT',
+    'ASCAT',
+    'ERS',
+  ],
+  providers: [
+    ee.producer_provider('pkurelab', 'https://pkurelab.github.io'),
+    ee.host_provider(self_ee_catalog_url),
+  ],
+  extent: ee.extent_global('1992-04-01T00:00:00Z', '2021-10-01T00:00:00Z'),
+  summaries: {
+    'gee:schema': [
+      {
+        name: 'Monthly radar signals',
+        description: 'The dataset is available from April 1992 to August 2021, with a spatial resolution of 8.9 km. To facilitate storage, the radar signals were scaled by a factor of 100 and converted to unsigned Int16.',
+        type: ee_const.var_type.int,
+      }，
+      {
+        name: 'Pearsonr',
+        description: 'Pearsonr between the monthly C-band and corrected Ku-band signals in the overlapping years.',
```

Comment:
please change "Pearsonr" to "Pearson correlation coefficient" in the description for clarity